en/docs/chapter_tree/binary_search_tree.md

```diff
@@ -31,25 +31,25 @@ Given a target node value `num`, one can search according to the properties of t
 === "<4>"
     ![bst_search_step4](binary_search_tree.assets/bst_search_step4.png)
 
-The search operation in a binary search tree works on the same principle as the binary search algorithm, eliminating half of the possibilities in each round. The number of loops is at most the height of the binary tree. When the binary tree is balanced, it uses $O(\log n)$ time. Example code is as follows:
+The search operation in a binary search tree works on the same principle as the binary search algorithm, eliminating half of the cases in each round. The number of loops is at most the height of the binary tree. When the binary tree is balanced, it uses $O(\log n)$ time. The example code is as follows:
 
 ```src
 [file]{binary_search_tree}-[class]{binary_search_tree}-[func]{search}
 ```
 
 ### Inserting a node
 
-Given an element `num` to be inserted, to maintain the property of the binary search tree "left subtree < root node < right subtree," the insertion operation proceeds as shown in the figure below.
+Given an element `num` to be inserted, to maintain the property of the binary search tree "left subtree < root node < right subtree", the insertion operation proceeds as shown in the figure below.
 
-1. **Finding the insertion position**: Similar to the search operation, start from the root node and loop downwards according to the size relationship between the current node value and `num` until passing through the leaf node (traversing to `None`) then exit the loop.
-2. **Insert the node at that position**: Initialize the node `num` and place it where `None` was.
+1. **Finding insertion position**: Similar to the search operation, start from the root node, loop downwards according to the size relationship between the current node value and `num`, until the leaf node is passed (traversed to `None`), then exit the loop.
+2. **Insert the node at this position**: Initialize the node `num` and place it at position `None`.
```

Comment:
"2. **Insert the node at this position**: Initialize the node `num` and place it at position `None`."
->
"2. **Insert the node at this position**: Initialize the node num and place it at the position of 'None'."

i still prefer the original as it seems a bit of clear and straightforward with the first of the changes:
"2. **Insert the node at this position**: Initialize the node `num` and place it where `None` was."